docs/news.rst

```diff
@@ -28,6 +28,8 @@ following backward-incompatible changes:
 -   The online documentation is moving to
     https://scrapy-zyte-smartproxy.readthedocs.io/
 
+-   New ``X-Crawlera-Client`` header support
+
```

Comment:
Adding it to this list makes it look like this was part of the rebranding.

What about adding an independent paragraph in these lines after the note below?

```
In addition to that, the ``X-Crawlera-Client`` header is now automatically included in all requests.
```

Reply:
Yes, looks good.
Updated.